content/about.html

```diff
@@ -135,44 +86,66 @@ <h2 class="no-link">
-                    audits for performance, accessibility, progressive web apps, and
-                    more.
+                    Audit your site with <a href="https://developers.google.com/web/tools/lighthouse/" target="_blank">Lighthouse</a> 
+                    to get guidance you can use to improve:
                   </p>
+                  <ul>
+                    <li>Performance</li>
+                    <li>Progressive Web App status</li>
+                    <li>Accessibility</li>
+                    <li>Use of best practices</li>
+                    <li>Search engine optimization</li>
+                  </ul>
                   <a href="/measure"
                       class="button button-flat gc-analytics-event"
                       data-category="web.dev"
                       data-label="about, explore measure"
-                      data-action="click">Learn More</a>
+                      data-action="click">Test my site</a>
                 </div>
                 <div class="whats-on__card">
                   <h3 class="no-link">Learn</h3>
                   <p>
-                    Organized around the core principles of what makes a better web,
-                    you'll find a comprehensive collection of actionable guides,
-                    tutorials, and codelabs to help you understand how to make your
-                    site better, and even practice doing it.
+                    Explore topics that matter for the modern web, 
+                    including areas like data security and load speed. 
+                    For each subject that interests you, expand your 
+                    knowledge and skills with:
                   </p>
+                  <ul>
+                    <li>Helpful guides</li>
+                    <li>Practical tutorials</li>
+                    <li>Codelabs for testing</li>
+                  </ul>
                   <a href="/learn"
                       class="button button-flat gc-analytics-event"
                       data-category="web.dev"
                       data-label="about, explore learn"
-                      data-action="click">Learn More</a>
+                      data-action="click">Keep learning</a>
                 </div>
               </div>
+              <h2 class="no-link">
+                Reaching users on the mobile web
+              </h2>
+              <div class="video-wrapper">
+                <iframe class="devsite-embedded-youtube-video" data-video-id="3bKMnYJzrqI"
+                        data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+                </iframe>
+              </div>
+              <p>
+                At first, Pinterest's mobile site pushed all their users to download their app.
+              </p>
+              <p>
+                Then, they invested in building a modern web experience. With a new focus on speed 
```

Comment:
Connect Then sentence to first paragraph, and start new second paragraph with 'With a new focus..." It makes more sense and looks better.